app/assets/javascripts/sumofus/backbone/form_methods.js

```diff
@@ -64,6 +65,32 @@ const FormMethods = {
     this.$(`.${plugin_type}-bar__welcome-name`).text(member.welcome_name);
     this.$(`.${plugin_type}-bar__welcome-text`).removeClass('hidden-irrelevant');
   },
+
+  insertActionKitId(form_type, urlParams) {
+    let $form;
+    if(form_type == 'petition') {
+      $form = $('.petition-bar__main').find('form')[0];
+    } else if(form_type == 'fundraiser') {
+      $form = $('.fundraiser-bar__step-panel').find('form')[0]
+    }
+
+    if(urlParams && 'akid' in urlParams) {
+      if($form) {
+        $('<input>').attr({
+          type: 'hidden',
+          name: 'actionkit_user_id',
+          value: urlParams['akid']
+        }).appendTo($form);
+      }
+    }
+  },
+
+  renameActionKitIdToReferringId() {
+    let $action_kit_hidden = $('input[name=actionkit_user_id]');
```

Comment:
I've always seen this type of selector with quotes around the name, like `$('input[name="actionkit_user_id"]');` Not sure why, does it work like this?